README.md

```diff
@@ -142,9 +142,14 @@ Complete all of these tasks on your local home machine.
 
         sudo pip install dopy==0.3.5
   * Google
+<p>
```

Comment:
This seems to cause an issue where the alignment of subsequent bullets is now off:
![offset screenshot](https://binaryparadox.net/d/31691b64-09db-4513-9c91-6a9dcef5eb93.jpg)

Reply:
Gahhh. Lemme look at that again.

Reply:
Thanks - sorry about all the markdown hassle!